givenergy_modbus/model/inverter.py

```diff
@@ -33,6 +33,20 @@ class Model(StrEnum):
     def _missing_(cls, value):
         """Pick model from the first digit of the device type code."""
         return cls(value[0])
+    
+    @classmethod
+    def add_regs(cls, value):
+        """Return possible additional registers."""
+        regs={
+            '2': ([],[180,240,300,360]),    #Hybrid
```

Comment:
Hmm.. they're all very different, aren't they. How interesting...

I've been playing with the notion of using python sets to express register sets. See https://github.com/divenal/givenergy-modbus-async/commit/ea41f0c92d6ce16ea7b9bf3bd4656be28b318984  (still work in progress)

Are these all possible registers that we need to probe for even after detecting the model type, or once the model has been established, are all these registers known to be there ?  For LV hybrid inverters, there's certainly a detail that later generations have more registers than older generations, so there may still be a need to probe to see what's available ?

Reply:
This set here is the additional registers which should be probed for each model type. For most they will always exist, for others they must be tested using detect_plant. This list gives the starting point then detect_plant confirms if they exists (which is based on firmware version in reality)